x509/bash.go

```diff
@@ -17,25 +17,26 @@ func run(name string, arg ...string) string {
 	return string(out)
 }
 
-func CreateFactoryCa(ou string) string {
+func CreateFactoryCa(storage KeyStorage, ou string) string {
```

Comment:
Are we planning to switch from the bash to go approach for the non-HSM use-case too?

Reply:
@mike-sul yes.

An intent of this PR was to enable the Golang approach for all use cases.
The HSM support was the only missing part to do that.

Reply:
Why the script is called at line #22 then?

Reply:
There will be a separate PR where the [go tags here](https://github.com/foundriesio/fioctl/blob/main/x509/bash.go#L1) disable Bash implementation altogether.
I already created such PR, but it is temporarily closed: https://github.com/foundriesio/fioctl/pull/267